examples/workflow/generic/simple_subworkflow_data/simple.ini

```diff
@@ -0,0 +1,9 @@
+[workflow]
+
+[executables]
+exe1 = /usr/bin/cp
```

Comment:
It's worth pointing out https://pegasus.isi.edu/documentation/manpages/pegasus-keg.html for doing stuff like this.

Reply:
Ah, right, I forgot about that. I guess that would have made a lot of sense to use here. We can always switch over if we add a few more complicated tests here. At least we have a place now to put in some simple PyCBC workflow tests, so hopefully we can cover our bases better and maybe expose a few of them as examples in documentation at some point.

Reply:
I somehow suspect that you and I and a few at the AEI are the only people who might use the pycbc workflow stuff for *convenience* and that's largely a lack of documentation I think.